backend/entities/Project.cs

```diff
@@ -235,6 +257,11 @@ public class Project : BaseEntity
         /// get/set - Record the interest component.
         /// </summary>
         public decimal InterestComponent { get; set; }
+
+        /// <summary>
+        /// get/set - Amount offered during SPL
+        /// </summary>
+        public decimal OfferAmount { get; set; }
```

Comment:
Where you planning to move this to the agency response table later?

Reply:
Also thinking this occurs once per project.

Reply:
this one would make sense to add I think